docs/designers-developers/developers/tutorials/create-block/submitting-to-block-directory.md

```diff
@@ -0,0 +1,86 @@
+#Share your Gutenberg Block with the World
+So you’ve created an awesome block have you? Care to share? Here is some basic information on how to submit a block to the Block Directory.
+
+**Contents**:
+1. Help users understand your block
+2. Analyze your plugin
+3. Zip & Submit
+
+
+## Step 1: Help users understand your block
+Providing straightforward, easy to understand block information is important to the block directory and our end users.
+
+**Guidelines**:
+
+- Name your block based on what it does
+- Clearly describe your block
+- Add Keywords for all contexts
+- Choose the right category
+
+### Name your block based on what it does
+Users typically search the block directory within gutenberg and do so in the context of a task. For example, when building their post, a user may search the Block Directory for an “image gallery”. Naming your block accordingly will help the Block Directory surface it when it's needed.
+
+**Not So Good**: WebTeam5 Image Works
+**Good**: Responsive Image Slider by WebTeam5
+
+**Question: What happens when there are multiple blocks with similar names?**
+Try your best to make your block's name functional and unique to make it stand out. Look for applicable synonyms or include a prefix if necessary.
+
+
+### Clearly describe your block
+The description really helps to communicate what your block does.The quicker a user understands how your block will help them, the more likely it is a user will use your block. Users will be reading your block's description within Gutenberg where space can be limited. Try to keep it short and concise.
+
+**Not So Good**: The best way to show images on your website using jQuery and CSS.
+**Good**: An easy to use, responsive, Image gallery block. 
+
+**Tip**: It’s not about marketing your block, in fact we want to avoid marketing in blocks. You can read more about it in the [plugin guidelines]. Stick to being as clear as you can. The block directory will provide metrics to let users know how awesome your block is!
+
+### Add Keywords for broader context
+Keywords add extra context to your block and make it more likely to be found in the inserter. 
+
+Examples for an Image Slider block:
+- slider
+- carousel
+- gallery
+
+[Read more](https://github.com/WordPress/gutenberg/blob/master/docs/rfc/block-registration.md#keyword) about keywords.
+
+### Choose the right category
+Gutenberg allows you to indicate which category your block fits in. Choosing the right category makes it easy for users to find in the Gutenberg menu.
```

Comment:
The Block Editor allows you to indicate the category your block belongs in, making it easier for users to locate your block in the menu.